src/pages/resultsView/cnSegments/CNSegments.tsx

```diff
@@ -51,9 +51,15 @@ export default class CNSegments extends React.Component<{ store: ResultsViewPage
         return generateSegmentFeatures(segments);
     }
 
+    @computed get genome() {
+        const study = this.props.store.studies.result?
+            this.props.store.studies.result[0]: undefined;
+        return study? study.referenceGenome: "hg19";
```

Comment:
maybe create a constant like `DEFAULT_GENOME`

Reply:
good idea.

Reply:
defined DEFAULT_GENOME in IGVUtils file.